src/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5239,7 +5239,7 @@ nuget update -Self</value>
   <data name="CommandFallbackSourceDescription" xml:space="preserve">
     <value>A list of packages sources to use as fallbacks for this command.</value>
   </data>
-  <data name="RestoreCommandMsBuildPath" xml:space="preserve">
-    <value>Path to MSBuild, defaults to '%ProgramFiles%\MSBuild\14.0\Bin\MSBuild.exe.</value>
+  <data name="CommandMSBuildVersion" xml:space="preserve">
+    <value>Specifies the version of MSBuild to be used with thise command. E.g. "14". If not specified, nuget will use the msbuild found in PATH. If msbuild is not in PATH, then MSBuild version 4.0 will be used.</value>
```

Comment:
How about this?

Specifies the version of MSBuild to be used with this command. For example, "14" for MSBuild version 14.0. If not specified, the MSBuild in PATH, or MSBuild version 4.0 if PATH does not contain MSBbuild, will be used.